README.md

```diff
@@ -1 +1,14 @@
-# cocoapod-specs
\ No newline at end of file
+# The Bitmovin CocoaPods Repo
+
+This repository contains all public CocoaPods specifications provided by [Bitmovin](https://bitmovin.com).
+
+## Using
+
+1. Add `source 'https://github.com/bitmovin/cocoapod-specs.git'` to your `Podfile`.
+1. Run `pod repo update` to add the newly added source.
+1. Add the desired pod (e.g. `pod 'BitmovinPlayer', '3.0.0'`) to your `Podfile`.
+1. Install the pod using `pod install`.
```

Comment:
Why don't we simply suggest using `pod install --repo-update`? Would remove one step 🙂 

```suggestion
1. Add the desired pod (e.g. `pod 'BitmovinPlayer', '3.0.0'`) to your `Podfile`.
1. Update the newly added source and install the pod using `pod install --repo-update`.
```

Reply:
True but `pod install --repo-update` is overkill if you don't need to update the repo. Would take way longer than without. Usually, one using cocoapods should know when to need the `repo update` step.

_Those steps are copied from the public sample repo_

Reply:
Okay, fine for me 👍